tools/vipps-ecom-api-postman-collection.json

```diff
@@ -45,11 +45,6 @@
 								"value": "application/x-www-form-urlencoded",
 								"type": "text"
 							},
-							{
```

Comment:
I added these here, but that turned out to be wrong. I now know how to run the postman collections, so I will be able to test them next time I fiddle with them.